_pages/en_US/troubleshooting.txt

```diff
@@ -505,6 +523,19 @@ The steps below can be attempted in any order, but are listed from least to most
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>My device does not show any of my apps on the HOME Menu</u></summary>
+
+This could be caused by various reasons, but mostly because your SD card is not being read by the system
+You can check if your SD is not being read by holding SELECT on boot and checking the yellow text on the bottom screen; If it says "Booted from CTRNAND via B9S", then your system is booting from the internal memory and not from the SD card.
```

Comment:
use lowercase after semicolon, but a period would work fine here too

Reply:
also, this could indicate that boot.firm is missing from the sd card for whatever reason

Reply:
stylistically speaking, bullet points do not usually use a period at the end